src/variant.cpp

```diff
@@ -418,6 +418,51 @@ namespace {
         v->stalemateValue = VALUE_MATE;
         return v;
     }
+	
```

Comment:
If you can find reliable sources for descriptions of (some of) the variants, preferably wikipedia, chessvariants.com, etc., it would be nice to put the links to them in comments for each variant, similar to how it is done for some other variants. It isn't a requirement though, just nice to have in case it is available.